org.lflang/src/org/lflang/federated/generator/FedGenerator.java

```diff
@@ -165,6 +166,21 @@ public boolean doGenerate(Resource resource, LFGeneratorContext context) throws
         return false;
     }
 
+    /**
+     * Check if a clean was requested from the standalone compiler and perform
+     * the clean step.
+     * @param context
+     */
+    protected void cleanIfNeeded(LFGeneratorContext context) {
+        if (context.getArgs().containsKey("clean")) {
```

Comment:
`LFGenerator.BuildParm.CLEAN.getKey()`

Reply:
I agree; this type of code appears to be all over the place (https://github.com/search?q=repo%3Alf-lang%2Flingua-franca+getArgs%28%29.containsKey&type=code); unsure if anyone's tackling this, otherwise I can change them at once later in a PR to ensure consistency

Reply:
Let's at least fix this one before we merge.